### ISSUE TITLE
I also can not reproduce the results reported in the paper

### ISSUE THREAD

**Issue by hugddygff** [zhanjunlang/Span_OIE#5]
and I doubt the realness of the paper

Can you provide pretrained model which can give the results reported in the paper.

**Comment by 131250208** [comment]
> and I doubt the realness of the paper
> 
> Can you provide pretrained model which can give the results reported in the paper.

Could you upload your results?

**Comment by hugddygff** [comment]
> > and I doubt the realness of the paper
> > Can you provide pretrained model which can give the results reported in the paper.
> 
> Could you upload your results?

I finally get the results in the paper. But I get such results in the first three batchs, so I think the training is not stable.

Usually we get the best results after many iterations.

**Comment by 131250208** [comment]
> > > and I doubt the realness of the paper
> > > Can you provide pretrained model which can give the results reported in the paper.
> > 
> > 
> > Could you upload your results?
> 
> I finally get the results in the paper. But I get such results in the first three batchs, so I think the training is not stable.
> 
> Usually we get the best results after many iterations.

Three batches? Do you mean the results on the whole test set? You got the results even without finishing the first epoch? It is amazing. Could you share the parameters you set?

**Comment by hugddygff** [comment]
> > > > and I doubt the realness of the paper
> > > > Can you provide pretrained model which can give the results reported in the paper.
> > > 
> > > 
> > > Could you upload your results?
> > 
> > 
> > I finally get the results in the paper. But I get such results in the first three batchs, so I think the training is not stable.
> > Usually we get the best results after many iterations.
> 
> Three batches? Do you mean the results on the whole test set? You got the results even without finishing the first epoch? It is amazing. Could you share the parameters you set?

three epoches, sorry, I make a mistake.
